ansible/provision.yml

```diff
@@ -0,0 +1,143 @@
+---
+- hosts: tildes_server
+  become: yes
+  
+  tasks:
+    - name: Allow SSH through the firewall
+      ufw:
+        rule: allow
+        port: ssh
+        proto: tcp
+
+    - name: Allow HTTP through the firewall
+      ufw:
+        rule: allow
+        port: http
+        proto: tcp
+
+    - name: Allow HTTPS through the firewall
+      ufw:
+        rule: allow
+        port: https
+        proto: tcp
+
+    - name: Turn on the firewall
+      ufw:
+        state: enabled
+        policy: deny
+
+    - name: Add tildes user
+      user:
+        name: tildes
+
+    - name: Add nginx user
+      user:
+        name: nginx
+        create_home: no
+        system: yes
+
+    - name: Copy the tildes source to the remote server
```

Comment:
Instead of copying files from the local system, we could also pull them directly from github: https://docs.ansible.com/ansible/latest/modules/git_module.html

Reply:
I think I like copying them from the local system better, because that makes it a *lot* easier to deploy a chosen version. Also you have to get the repo anyway in order to run this script, it's kind of weird if the script installs a different version than the one you got.

Reply:
Sounds good.